spec-dtslint/util/pipe-spec.ts

```diff
@@ -75,3 +75,10 @@ it('should enforce types for the 8th argument', () => {
 it('should enforce types for the 9th argument', () => {
   const o = pipe(a('0', '1'), a('1', '2'), a('2', '3'), a('3', '4'), a('4', '5'), a('5', '6'), a('6', '7'), a('7', '8'), a('#', '9')); // $ExpectError
 });
+
+it('should inherit typings from source Observable ', () => {
```

Comment:
I wouldn't use the word 'inherit' here - it has a specific meaning. I'd say that it should return a generic `OperatorFunction`.